README.rst

```diff
@@ -1,9 +1,9 @@
+.. image:: doc/source/_static/scipyshiny_small.png
+  :target: https://docs.scipy.org/doc/scipy/reference/
```

Comment:
This is very large, could you make it a couple of times smaller?

Reply:
I can and I will try to use also `:align: left`

Reply:
Apparently I need raw html here.

Reply:
It works 😃